waspc/data/Generator/templates/server/src/routes/auth/utils.js

```diff
@@ -0,0 +1,10 @@
+{{={= =}=}}
+
+import prisma from '../../dbClient.js'
+
+export const contextWithUserEntity = { entities: { {= userEntityUpper =}: prisma.{= userEntityLower =} } }
```

Comment:
I would probably format this to multiple lines to improve readability (especially because there's a lot of template stuff).

Reply:
Good point, will do!